https://issues.redhat.com/browse/ACM-17599 -- Add Building images concept sections (#7604)
open-cluster-management/rhacm-docs: edge_manager/edge_mgr_images.adoc

The {rhem} focuses on image-based Linux operating systems that run bootable container image (`bootc`).
For more information, see https://bootc-dev.github.io/bootc/[bootc].

*Important:* The `bootc` tool does not update package-based operating systems.

For more information, see the _Additional resources_ section.

[#images-process]
== The image building process

At a high level, the image building process works as follows:

. Choose a base `bootc` operating system image, such as a Fedora, CentOS, or RHEL image.
. Create a container file that layers the following items onto the base `bootc` image:
* The {rhem} agent and configuration.
* Optional: Any drivers specific to your target deployment environment.
* Optional: Host configuration, for example certificate authority bundles, and application workloads that are common to all deployments.
. Build, publish, and sign a `bootc` operating system image using `podman` and `skopeo`.
. Create an operating system disk image by using `bootc-image-builder`.
. Build, publish, and sign an operating system disk image using `skopeo`.

> *Note:* The operating system disk image contains partitions, volumes, the file system as well as the initial `bootc` image. The operating system disk image only needs to be created once, during provisioning.

Reviewer: ```suggestion
*Note:* The operating system disk image contains partitions, volumes, the file system, and the initial `bootc` image. The operating system disk image only needs to be created once, during provisioning.
```
